src/public/sql/initial.sql

```diff
@@ -0,0 +1,108 @@
+use laraveldb;
+
+-- for testing - drop all the tables
+drop table if exists user, user_session, role, ui_component, m_role_component;
+
+-- user related
+CREATE TABLE IF NOT EXISTS user (
+    id MEDIUMINT NOT NULL AUTO_INCREMENT,
+    username VARCHAR(20) NOT NULL, 
+    password VARCHAR(20),
+    role_id MEDIUMINT,
+    first_name VARCHAR(20), 
+    middle_name VARCHAR(20),
+    last_name VARCHAR(20), 
+    email VARCHAR(20),
```

Comment:
We should probably use VARCHAR(255) to account for some longer emails

Reply:
Yes, right, I will change that. Thanks for point out.

Reply:
Can we get rid of username and middle name, and possibly ID if it isn't being used anywhere else? Email could be our unique identifier.